### ISSUE TITLE
No confidence intervals for lmer model from ggpredict

### ISSUE THREAD

**Issue by Osmia86** [strengejacke/ggeffects#126]
**How can I bring ggpredict to calculate confidence intervals?**
Hi! I'm still relatively new to R. 
I was excited to find that you can add raw data points to a plot of an e.g. interactive effect using ggpredict and the plot function. Unfortunately, ggpredict isn't calculating any confidence intervals. I would appreciate any help!
I'm using the following: R-version 3.6.3, lme4_1.1-21, ggeffects_0.14.2
This is the model:
```fs <- lmer(speed ~ width * cond + (1|cond/b), data=fs)```
speed is numerical, width consists of integer numbers, cond is a factor with 2 levels and b is also a factor with 14 levels.
This is how I tried to get a plot of the interactive effect between width and condition
```dat <- ggpredict(fs, terms = c("width","cond"),type="re",interval = "confidence",back.transform = TRUE)```
```plot(dat, colors="bw", rawdata = TRUE,facet=T,ci.style = c("ribbon"))```
This gives me wonderful graph. However, it doesn't contain any confidence intervals, as they are already missing from the ggpredict function.
Here is an output for the condition "acryl" when calling dat:
```dat```
```Predicted values of speed```
```x = width```
```cond = acryl```
```  x | Predicted```
```---------------```
```205 |      2.85```
```305 |      2.77```
```400 |      2.69```
```500 |      2.61```
```600 |      2.53```
```795 |      2.37```
As you can see, there are no values for the confidence intervals. So, what am I doing wrong here?
Please let me know if you need more info!
Here is also the data I'm using:
[200313_FlightSpeed.txt](https://github.com/strengejacke/ggeffects/files/4362492/200313_FlightSpeed.txt)

**Comment by strengejacke** [comment]
See [this vignette](https://strengejacke.github.io/ggeffects/articles/introduction_randomeffects.html#marginal-effects-for-each-level-of-random-effects). This is not a bug per se, but calculating confidence intervals for predictions of random effect groups is not straightforward, that's why these are not yet implemented. If you know of any package that can do this, let me know, so I can try to find out how to address this issue...

**Comment by Osmia86** [comment]
Dear Daniel,
Thank you very much for the quick response and the help. I've looked into the vignette and I think I get it now!
Cheers,
Olga

**Comment by jukicivan** [comment]
Dear @strengejacke AND @Osmia86 , 

I apologise for jumping into this thread, but I though it would be more convenient to post my question here since it's very related.

Namely, I have the same issue with missing confidence intervals after using ggpredict on the lmer-generated model. However, I actually used "type = fe" (i.e., default behaviour of the function) because I'd be fine with predictions on the population-level that do not account for the random effect variances. Unfortunately, while doing so, I realised that I'm missing confidence intervals for some of the groups.

Predictor 2 = RTF80
Predictor 3 = 38.4

Predictor1.    Predicted.    95% CI
female          7.63              [5.40, 9.87]
male             6.63 

In one instance, I'm missing CI's for both groups (i.e., levels of the predictor 1). I'm having three terms as follows: 

`ggpredict(mod, terms = c("Pred1", "Pred2", "Pred3"), type = "fe", interval = "confidence")`

I'm not sure why this happens. Importantly, not using predictor 3 in the `ggpredict` normally calculates confidence intervals for the other two predictors. In addition, using a different predictor 3 from the one that I'm specifically interested in also leads to confidence intervals calculation. However, using the predictor 3 that I'm interested in leads to "prediction intervals" calculation if I specify type = "fe". So, my solution for now is to go with prediction intervals, but I'm very curious to find out why is this the case? I have no missing values whatsoever for this predictor 3 of interest. 

Thank you for your time!
